libsolidity/codegen/ir/IRGenerationContext.cpp

```diff
@@ -170,6 +170,42 @@ void IRGenerationContext::internalFunctionAccessed(Expression const& _expression
 	}
 }
 
+pair<u256, unsigned> IRGenerationContext::storageLocationOfStateVariable(VariableDeclaration const& _varDecl) const
+{
+	solAssert(isStateVariable(_varDecl), "");
+	return m_stateVariables.at(&_varDecl);
+}
+
+pair<string, string> IRGenerationContext::storageLocationOfLocalVariable(VariableDeclaration const& _varDecl) const
```

Comment:
Wouldn't it be better to inline this function?

Reply:
:+1: for inlining